source/hardware/controllers/gemini_g4v.rst

```diff
@@ -0,0 +1,250 @@
+
+The deck's :hwlabel:`CUP` buttons will start playback from the beginning of the track.
+
+Jog Wheels
+----------
+
+Pressing the top of the Jog Wheel will activate scratching mode mode. It will stop playing.
+
+Turning the jog wheel will move the track play head, for scratching or fine placement.
+
+Turning the jog wheel while pressing shift will change the track play head fast, for fast search.
+
+When the jog wheel top is released, the jog wheel will stay in scratching mode until it stops moving, for spin backs or similar actions
+
+If the deck was in slip mode before entering in scratching mode, the play head will move to the position it will have been if no manipulation happened, useful for scratching without losing the beat
+
+Turning the jog wheel without pressing the top will temporarily change the tempo, for nudging.
+
+Performance Pads
+----------------
+
+Above the wheel on each deck is a set of 8 performance pads (top pads numbered 1-4 and bottom pads numbered 5-8 for the remainder of this manual page).
+
+The functionality of the performance pads is determined by the pads mode: Hot Cue, Auto Loop, Sample, Loop Roll, manual Loop and Beat Jump.
+
+You can can change the mode by using the mode buttons as indicated.
+
+Hot Cue Mode
+^^^^^^^^^^^^
+
+To enable Hot Cue mode, press the :hwlabel:`Hot Cue` button.
+When the performance pads are in Hot Cue mode, the Hot Cue button will be lit up.
+
+Pressing a performance pad when in Hot Cue mode will control the corresponding numbered Hot Cue (Performance Pad 1 controls Hot Cue 1, etc) on the virtual deck.
+If the corresponding Hot Cue isn't set, pressing the performance pad will set a hot cue at that point.
+If the corresponding Hot Cue is set, pressing the performance pad will move playback to that hot cue.
+If a hot cue is set and it's corresponding performance pad is pressed while the deck's Shift button is being held, the hot cue will be deleted.
+
+Sample Mode
+^^^^^^^^^^^
+
+To enable Sample mode, press the :hwlabel:`Sample` button.
+When the performance pads are in Sample mode, the Sample button will be lit up.
+It will show the sample decks if the skin has support for them.
+
+Pads will be lit up if they have a track loaded.
+Each deck control 8 samples, deck 1 controls samples 1 to 8, deck 2 controls samples 9 to 16, an so on.
+
+The :hwlabel:`Sample Volume` knobs control the volume for all the samples.
+
+Auto Loop Mode
+^^^^^^^^^^^^^^
+
+To enable looping mode, press the :hwlabel:`Auto Loop` button.
+When the performance pads are in Auto Loop mode, the Auto Loop button will be lit up.
+
+Pressing a pad starts a loop of the length indicated in the pad, the pad will lit up.
+Pressing the lit pad stop the loop.
+Pressing a different pad will stop the first pad and will Cstart a new loop.
+
+The :hwlabel:`Loop Move` knob shifts the loop position in the track.
+
+Loop Roll Mode
+^^^^^^^^^^^^^^
+
+To enable loop roll mode, press the :hwlabel:`Loop Roll` button.
```

Comment:
I don't see a LOOP ROLL knob labeled on the controller.

Reply:
It's the rightmost button under the pads.

Reply:
Okay, found it:https://i.pinimg.com/originals/1c/e0/c2/1ce0c2af2d3f455ca0cbca88c384c17a.jpg

Before, I used this image which has different labels for some reason. In that case some of my changes are probably wrong:
https://idmmag.com/wp-content/uploads/2013/01/Gemini-G4V.jpg